### PR TITLE
Employee CRUD Completed

### PR DESCRIPTION
**Pull request by Akhwandzada1**

### CODE REVIEW

File: resources/views/employees/form.blade.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<form action="@isset($employee) {{ route('employees.update', $employee->id) }} @else {{ route('employees.store') }} @endisset" class="form-validate is-alter" id="employee_form">
+    @isset($employee)
+        @method('PUT')
+    @endisset
+        @csrf
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    @isset($employee)
```
Comment (zeeshanmughaltherightsw) [inline resources/views/employees/form.blade.php]: Isset must be inside h5 tag

File: resources/views/employees/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+                <a href="#" class="btn btn-icon btn-trigger toggle-expand mr-n1" data-target="pageMenu"><em class="icon ni ni-menu-alt-r"></em></a>
+                <div class="toggle-expand-content" data-content="pageMenu">
+                    <ul class="nk-block-tools g-3">
+                        <li>
+                            <!-- <a href="{{ route('companies.create') }}" class="btn btn-white btn-outline-light">
+                                        <em class="icon ni ni-plus">
+                                        </em>
+                                        <span>Add Company</span>
+                                    </a> -->
+                        <li class="preview-item">
```
Comment (zeeshanmughaltherightsw) [inline resources/views/employees/index.blade.php:27]: Add permission on this button

File: resources/views/employees/form.blade.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            </div>
+        </div>
+        <div class="form-group">
+            <label class="form-label required" for="email-address">Last Name</label>
+            <div class="form-control-wrap">
+                <input type="text" class="form-control" id="last_name" value="@isset($employee){{ $employee->last_name }}@endisset" name="last_name" required>
+            </div>
+        </div>
+        <div class="form-group">
+            <label class="form-label required" for="phone-no">Email</label>
```
Comment (zeeshanmughaltherightsw) [inline resources/views/employees/form.blade.php]: Id for input and for attribute of label must be same

File: resources/views/employees/index.blade.php [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+                },
+                {
+                    name: 'action',
+                    data: 'action',
+                    orderable: false,
+                    searchable: false
+                }
+            ]
+        });
+        $('#employee_table_filter').css('padding-left', '63%');
```
Comment (zeeshanmughaltherightsw) [inline resources/views/employees/index.blade.php]: remove this

File: resources/views/employees/index.blade.php [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+                confirmButtonText: 'Yes, delete it!'
+            }).then(function(result) {
+                console.log(result);
+                if (result.value) {
+                    axios.delete(deleteUrl)
+                    .then(function (response){
+                        NioApp.Toast(response.data.message, 'success');
+                        $('#employee_table').DataTable().ajax.reload();
+                    }).catch(function (error){
+                        NioApp.Toast('Unable to delete company', 'error');
```
Comment (zeeshanmughaltherightsw) [inline resources/views/employees/index.blade.php]: Why is this constant here?

File: resources/views/employees/index.blade.php [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+                    })
+                }
+            });
+            e.preventDefault();
```
Comment (zeeshanmughaltherightsw) [inline resources/views/employees/index.blade.php]: remove this

**Review by zeeshanmughaltherightsw — CHANGES_REQUESTED** [review]

File: resources/views/employees/form.blade.php (hunk 1 above)
Comment (Akhwandzada1) [reply resources/views/employees/form.blade.php]: Done.

File: resources/views/employees/index.blade.php (hunk 2 above)
Comment (Akhwandzada1) [reply resources/views/employees/index.blade.php:27]: Done.

File: resources/views/employees/form.blade.php (hunk 3 above)
Comment (Akhwandzada1) [reply resources/views/employees/form.blade.php]: Done.

File: resources/views/employees/index.blade.php (hunk 4 above)
Comment (Akhwandzada1) [reply resources/views/employees/index.blade.php]: Done.

File: resources/views/employees/index.blade.php (hunk 5 above)
Comment (Akhwandzada1) [reply resources/views/employees/index.blade.php]: Put it for testing purposes. Removed & replaced dynamic string.

File: resources/views/employees/index.blade.php (hunk 6 above)
Comment (Akhwandzada1) [reply resources/views/employees/index.blade.php]: Removed.
